go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/google/go-cmp v0.5.6
 	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
 )
+
+require rsc.io/binaryregexp v0.2.0 // indirect
```

Comment:
nit: can you go mod tidy? (I should probably add that to CI later)

Reply:
done